### PR TITLE
code review practice

### PR DESCRIPTION
**Pull request by sheldonwafula**
In this pull request:
I have created HTML and CSS files.
I have added a descriptive readme file.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+I would like to thank my long time friend Mudasir for his
+endless support from the beginning of this project upto the end.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (sheldonwafula) [inline Readme.md:178]: The license link is not working. I would suggest you add .LICENSE to make your readme more professional.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            <img class="image" src="image/pexels-minenhle-shelembe-5985865.jpg" alt="Nelson Mandela">
+            <figcaption id="img-caption">Picture of Nelson Mandela</figcaption>
+        </div>
+        <div class="tribute-info" >
+            <p>About Nelson madela</p>
+            <a class="info" href="https://en.wikipedia.org/wiki/Nelson_Mandela" target="_blank">Read More about our hero Nelson Mandela</a>
+        </div>
+
+
+    </main>
```
Comment (sheldonwafula) [inline index.html:23]: I would suggest you align your code properly for easier reading, maintaining, and debugging.

**Review by sheldonwafula — COMMENTED** [review]
Good job so far.👏👏
However, I would suggest you make the required changes below.

**Review by braincee — APPROVED** [review]
Hi @sheldonwafula 👋  , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**